__tests__/FollowingButton-test.js

```diff
@@ -0,0 +1,68 @@
+import 'react-native';
+import React from 'react';
+import FollowingButton from '@screens/app/Components/FollowingButton/FollowingButton';
+import {shallow} from 'enzyme';
+import {Chip} from 'react-native-paper';
+import renderer from 'react-test-renderer';
+
+/** See FollowerButton-test for more notes on the tests! */
+
+test('Following Button Snapshot Test', () => {
+  const tree = renderer.create(<FollowingButton />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+/** This is currently testing the dummy placeholder code. However,
+ * it will need to be resturcutred to open up a new page that passes through an array of following.
+ */
+function testOnFollowingClicked(user_id) {
+  it('onFollowingClicked username is returned', () => {
+    const wrapper = shallow(<FollowingButton user_id={user_id} />);
+    const chipWrapper = wrapper.find(Chip);
+    const chipProps = chipWrapper.first().props();
+    const chipPressFunctionVal = chipProps.onPress();
+    console.log(chipPressFunctionVal);
+    expect(chipPressFunctionVal).toEqual(
+      'This is the following list of ' + user_id,
+    );
+  });
+}
+
+/** Check that the property values passed into the prop, appear on the prop. */
+function testProperPropsPassed(user_id, following_count) {
+  it('User ID value is the Following Button prop value', () => {
+    /** Does it matter if we pass in follower_count simulatanously? */
+    const wrapper = shallow(<FollowingButton user_id={user_id} />);
```

Comment:
Are we handling an empty user_id?

Reply:
We set prop types which warns us if a string is not entered